setup.py

```diff
@@ -28,6 +27,9 @@
     url='https://github.com/aws/chalice',
     packages=find_packages(exclude=['tests']),
     install_requires=install_requires,
+    extras_require={
+        'file-watcher': ['watchdog==0.8.3'],
```

Comment:
I'm wondering if there is a better name for this optional dependency? The name `file-watcher` is not too obvious as to what I am getting out of it in installing it since chalice already has a built in file watcher (without needing this optional dependency).

Reply:
How about `event-file-poller`?

Reply:
Yeah that sounds better to me